src/types/ConnectData.ts

```diff
@@ -50,8 +50,15 @@ export type WalletInfo = {
   name?: string;
 };
 
+export type AccountAttributes = {
```

Comment:
I'm curious why you chose to define the type as an array instead of a single instance of an attribute and then use array syntax where it's used. Seems like it would be clearer that way. My initial assumption was that `AccountAttributes` was an object, not an array of objects.

Reply:
Agreed, will change. No reason for it to be the way it is now.